src/utils/id.servise.js

```diff
@@ -0,0 +1,7 @@
+function createId(array) {
```

Comment:
There is a typo in the file name: 'id.servise.js' should likely be 'id.service.js'. This will help maintain consistency and prevent potential confusion.